src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd

```diff
@@ -62,6 +63,10 @@
         <xsd:attribute name="path" type="xsd:string" />
     </xsd:complexType>
 
+    <xsd:complexType name="links">
+        <xsd:attribute name="enabled" type="xsd:boolean" />
```

Comment:
maybe something to "fix" by adding a new "boolean" type, borrowed from the XSD of the DI XmlFileLoader?